packages/ui/components/input-file/translations/bg.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  ALLOWED_FILE_SIZE: 'Моля, качете файл с размер, не по-голям от {params}.',
```

Comment:
Is there a reason we have a different naming convention (instead of camelCased) here?

Reply:
Sole reason that the codebase i used had this already. I will turn it to camelCase :)